Platform/QemuBoardPkg/Script/TestCases/firmware_update.py

```diff
@@ -257,8 +257,9 @@ def main():
     cmd = [ sys.executable,
             'BootloaderCorePkg/Tools/GenCapsuleFirmware.py',
             '-p',  'BIOS', bios_img,
-            '-k',  '../SblKeys/FirmwareUpdateTestKey_Priv_RSA2048.pem',
-            '-o',  '%s/FwuImage.bin' % fwu_dir
+            '-k',  '../SblKeys/FirmwareUpdateTestKey_Priv_RSA3072.pem',
+            '-o',  '%s/FwuImage.bin' % fwu_dir,
+            '-a',  'SHA2_384'

```

Comment:
can;'t this be determined by the key type ?
EX: 3K => SHA384  2K=>SHA256

Reply:
Our signing interface is generic to support multiple key types and signing hash alg  combinations.
If we restrict, 2k --> SHA256 and 3K--> SHA384, we could adjust this in respective tools.
This adjust is getting done in container due to auth types